xbmc/music/MusicInfoLoader.cpp

```diff
@@ -107,14 +107,15 @@ bool CMusicInfoLoader::LoadAdditionalTagInfo(CFileItem* pItem)
 
   CLog::Log(LOGDEBUG, "Loading additional tag info for file %s", path.c_str());
 
-  // we load up the actual tag for this file
-  unique_ptr<IMusicInfoTagLoader> pLoader (CMusicInfoTagLoaderFactory::CreateLoader(*pItem));
+  // we load up the actual tag for this file in order to
+  // fetch the lyrics and add it to the current music info tag
+  CFileItemPtr item(new CFileItem(path, false));
```

Comment:
Maybe name this `tempItem` to make it obvious that i'ts just a temporary? And it also doesn't really need to be a `CFileItemPtr`. A simple `CFileItem` should be enough.